test/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Needed to specify that in this repository, the environment is different (it is using mocha)